x/move/common_test.go

```diff
@@ -113,7 +113,7 @@ func executeMsgs(t *testing.T, app *initiaapp.InitiaApp, msgs []sdk.Msg, account
 
 func decToVmArgument(t *testing.T, val math.LegacyDec) []byte {
 	// big-endian bytes
-	bz := val.BigInt().Bytes()
+	bz := slices.Clone(val.BigInt().Bytes())
```

Comment:
`BigInt()` returns a copy of instance and `Bytes()` also returns newly assigned it.